src/main/scala/io/iohk/ethereum/jsonrpc/server/http/SecureJsonRpcHttpServer.scala

```diff
@@ -25,6 +28,14 @@ class JsonRpcHttpsServer(
     extends JsonRpcHttpServer
     with Logger {
 
+  override val ipTrackingEnabled: Boolean = config.ipTrackingEnabled
```

Comment:
(ditto InsecureJsonRpcHttpServer)
With the proposal of JsonRpcHttpServer, we need to add the next val, and remove the overridden values:

val config: JsonRpcHttpServerConfig